webview-ui/src/components/ChatList.tsx

```diff
@@ -18,18 +20,47 @@ export const ChatList = ({ chatList, changeActiveChat }: ChatListProps) => {
     changeActiveChat(conversation);
   };
 
+  const listOfChats = chatList.map((conversation) => {
+    return (
+      <div className="convo">
+        <div
+          onClick={() => handleOnClick(conversation)}
+          key={conversation.id}
+          className="convo-id"
+        >
+          {conversation.id}
+        </div>
+        {/* TODO: add a function to delete a single conversation to make button work */}
+        <VSCodeButton
+          appearance="icon"
+          aria-label="Delete conversation"
+          title="Delete conversation"
+          onClick={() => extensionMessenger.deleteConversation(conversation.id)}
+        >
+          <i className="codicon codicon-remove"></i>
```

Comment:
I might suggest using the "trash" icon instead of the minus/remove, but either probably work